frontend/app/src/pages/main/workflow-runs/$run/v2components/view-toggle.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@/components/ui/button';
+import { WorkflowRunShape } from '@/lib/api';
+import { preferredWorkflowRunViewAtom } from '@/lib/atoms';
+import { useAtom } from 'jotai';
+import { BiExitFullscreen, BiExpand } from 'react-icons/bi';
+
+type Props = {
+  shape: WorkflowRunShape;
+};
+
+export const ViewToggle: React.FC<Props> = ({ shape }) => {
+  const [view, setView] = useAtom(preferredWorkflowRunViewAtom);
+  const otherView = view === 'graph' ? 'minimap' : 'graph';
+
+  // only render if there are at least two dependent steps, otherwise the view toggle is not needed
+  if (!hasChildSteps(shape)) {
```

Comment:
are we storing the view in local store? if yes, will we still need the toggle if its set on run A but run B matches this condition?

Reply:
disregard, ran the PR and see the behavior